exo-go/src/os_helpers/index.go

```diff
@@ -0,0 +1,72 @@
+package osHelpers
+
+import (
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+
+	"github.com/tmrts/boilr/pkg/util/osutil"
+)
+
+// IsEmpty returns true if dirPath is an empty directory, and false otherwise
+func IsEmpty(dirPath string) bool {
+	f, err := os.Open(dirPath)
+	if err != nil {
+		return false
+	}
+	_, err = f.Readdir(1)
+	return err == io.EOF
+}
+
+// IsValidTemplateDir returns true if the directory templateDir is a valid
+// boilr template directory
+func IsValidTemplateDir(templateDir string) bool {
+	return FileExists(path.Join(templateDir, "project.json")) && DirectoryExists(path.Join(templateDir, "template"))
+}
```

Comment:
I know I moved a lot of the methods around, but can we move this template_helpers since this is more about templates then a generic os helper?